keras_rcnn/heads/resnet.py

```diff
@@ -0,0 +1,96 @@
+import keras
+
+
+def ResHead(classes, mask=False):
+    """Resnet heads as in Mask R-CNN."""
+    def f(x):
+        if keras.backend.image_data_format() == "channels_last":
+            channel_axis = 3
+        else:
+            channel_axis = 1
+
+        y = keras.layers.TimeDistributed(
```

Comment:
Good question. 😆 

A quick method might be iterating over model.layers then pop each layer and append TimeDistributed.